upgrade_tests/test_existance_relations/cli/test_gpg.py

```diff
@@ -30,7 +29,6 @@
 # Tests
 
 
-@dont_run_to_upgrade(['6.8', '6.9', '6.10', '6.11'])
```

Comment:
Couldn't find any useful information why this started to be skipped since 6.7
Giving this test a chance as I see no reason why gpg keys could change between versions